docker/Dockerfile.pdo-dev

```diff
@@ -91,6 +91,7 @@ RUN apt-get update \
     libsecp256k1-dev \
     libssl-dev \
     libtool \
+    tinyscheme \
```

Comment:
If tinyscheme is now installed here, is the `tinyscheme-1.41` wget and build part below needed?

Reply:
no. i will remove them.